ipu-plugin/pkg/ipuplugin/ipuplugin.go

```diff
@@ -81,6 +82,8 @@ func (s *server) Run() error {
 	}
 
 	pb.RegisterBridgePortServiceServer(s.grpcSrvr, s)
+	dpu_api.RegisterVendorManagementServiceServer(s.grpcSrvr, NewVendorManagementService(s.servingHost, s.servingPort, s.mode))
```

Comment:
I don't think we can use same IP+port to start another server instance. It will always fail. 
Instead of creating separate server we should just register the new grpc service to existing server. We want to expose one single endpoint to the DPU daemon.

Reply:
This method registers another service with grpcServer 

```
func RegisterVendorManagementServiceServer(s grpc.ServiceRegistrar, srv VendorManagementServiceServer) {
	s.RegisterService(&VendorManagementService_ServiceDesc, srv)
}
```